File: app/src/main/java/net/tlalka/android/fiszki/model/dto/LanguagesDto.java
package net.tlalka.android.fiszki.model.dto;

import java.util.List;

import net.tlalka.android.fiszki.model.types.LanguageType;
import org.parceler.Parcel;

@Parcel
public class LanguagesDto {

    protected List<LanguageType> languages;

    public LanguagesDto() {
    }

    public LanguagesDto(List<LanguageType> languages) {
        this.languages = languages;
    }

    public List<LanguageType> getLanguages() {
        return languages;
    }
}

File: app/src/main/java/net/tlalka/android/fiszki/model/dto/LanguagesDto.kt
package net.tlalka.android.fiszki.model.dto

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import net.tlalka.android.fiszki.model.types.LanguageType

@Parcelize
data class LanguagesDto constructor(var languages: List<LanguageType>) : Parcelable